### PR TITLE
Update docs on setting up a development environment

### PR DESCRIPTION
**Pull request by shaneknapp**
just updating the docs a bit to make running this locally a little easier.

@yuvipanda

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -65,7 +65,7 @@ rules on your computer. We will now walk you through the steps to get going:
 
 1.  Install VirtualBox by [downloading and running an
     installer](https://www.virtualbox.org/wiki/Downloads), or
-    [Docker|https://www.docker.com/].
+    [Docker](https://www.docker.com/).
```
Comment (consideRatio) [inline CONTRIBUTING.md:68]: Ah from https://minikube.sigs.k8s.io/docs/drivers/ I understand that docker is available as a driver, and even preferred while virtualbox isn't preferred.

Have you tried getting kubespawner setup with minikube and docker as a driver to create the k8s cluster and verified that you could run the kubespawner test suite against it successfully?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -151,13 +152,13 @@ rules on your computer. We will now walk you through the steps to get going:
 
     ```sh
     npm install configurable-http-proxy
-    export PATH=$(pwd)/node_modules/.bin:$PATH
+    export PATH=/<path>/<to>/node_modules/.bin:$PATH
```
Comment (consideRatio) [inline CONTRIBUTING.md:155]: If following the command `npm install configurable-http-proxy`, the path to node_modules will be from the local directory as known by `$(pwd)`.

Due to that I think what we have already is quite good.

**Comment by consideRatio** [comment]
Thank you @shaneknapp !

**Comment by shaneknapp** [comment]
ugh thank YOU for following up on this!  Real Life[tm] is hard!  XD

**Pull request merged**